cdp_scrapers/README.md

```diff
@@ -0,0 +1,125 @@
+# Notes on writing LegistarScrapers
+
+## Filtering
+
+1. Unimportant `EventMinutesItem`
+
+
+`EventMinutesItem(minutes_item=MinutesItem(name="123", description="This meeting 
+also constitutes a meeting of the City Council, provided ...", ...), ...)`
+
+`minutes_item.description` comes from Legistar EventItem["EventItemTitle"]. Legistar 
+EventItems wil often contain unimportant information, as shown above, that we 
+want to exclude from ingesting into CDP. The `IGNORED_MINUTE_ITEMS` attribute in 
+`LegistarScraper` 
+defies a `List[str]` that we want to filter out. 
+`IGNORED_MINUTE_ITEMS` is used in `filter_event_minutes()`, case-insensitively.
+
+In your LegistarScraper-derived class' `__init__()`, overwrite or append to 
+`self.IGNORED_MINUTE_ITEMS`. e.g.
+
+```
+class MyScraper(LegistarScraper):
+    def __init__(self):
+        super().__init__("my_city")
+
+        self.IGNORED_MINUTE_ITEMS[MinutesItem].append("This meeting also constitutes a meeting")
+```
+
+With that filter in place, this entire `EventMinutesItem` becomes `None`.
+
+2. Empty models
+
+Consider another example such as
+
+`EventMinutesItem(decision=None, matter=None, minutes_item=None, supporting_files=[], ...)`
+
+which means nothing. `MIN_INGESTION_KEYS` is the LegistarScraper attribute used 
+to return `None` for such "empty" models. Like `IGNORED_MINUTE_ITEMS` it defines a `List[str]`, 
+per `IngestionModel` class. Here, each string in the `List[str]` is a key in the 
+corresponding model. For example, the base `LegistarScraper` class defines
+
+```
+self.MIN_INGESTION_KEYS = {
+    ...
+    EventMinutesItem: ["matter", "minutes_item"],
+    ...
+}
+```
```

Comment:
Long term it would be great to write a function to get the required attrs of an ingestion model based off of typing so that this can be programmitically generated instead of manually maintained but thats for later.

Reply:
Actually went ahead and did (at least a poor man's job of) this, since I was already fixing related code to use the correct required `IngestionModel` attributes. I'm going to do a more careful side-by-side debugging with Legistar data now. But Jackson whenever you can, can you also check out the head version and let me know what you think?

All the `IngestionModel` attributes without default values in the `class` definitions are `min_ingestion_keys` in `LegistarScraper` now. Took out my dumb liberal interpretation! Haha.

https://github.com/dphoria/cdp-scrapers/blob/5f7bab70a4436c48dc1150867805b87667a0fb19/cdp_scrapers/legistar_utils.py#L959

Reply:
This is super smart! Will give it a look in a bit.

Reply:
Oh also, if there is a more correct/pythonic way of what I did in `get_required_attrs()`, please do let me know.